example/heroic.example.yml

```diff
@@ -1,287 +1,48 @@
-#    # Maximum byte size of messages the consumer can have unacked.
-#    # @default 1000000000
-#    #maxOutstandingRequestBytes: 2000000000
-#    # Maxiumum size of a single incoming message, in bytes. 20MB is the highest allowed by the API.
-#    # @default 20971520
-#    #maxInboundMessageSize: 40960000
-#    # Amount of time to keep the PubSub connection alive, in seconds.
-#    # @default 300
-#    #keepAlive: 600
-
-## Aggregation cache.
-#cache:
-#  backend:
-#    ## In-memory based aggregation cache.
-#    #type: memory
-#    ## Cassandra based cache.
-#    #type: cassandra2
-#    #  # Keyspace.
-#    #  # @default "heroic"
-#    #  #keyspace: heroic
-#    #  # Seeds.
-#    #  # @default empty
-#    #  #seeds:
-#    #  #  - localhost
-#    #  # Max number of connections per host in cluster.
-#    #  #maxConnectionsPerHost: 50
-
-## Http client configuration.
-#client:
-#  # Number of threads to use for client requests.
-#  # @default 100
-#  #threads: 100
-#  # Max amounts of pending requests, if all client threads are occupied.
-#  # @default 100
-#  #queueSize: 100
-#  # Timeout in milliseconds for establishing a client connection.
-#  # @default 2000
-#  #connectTimeout: 2000
-#  # Timeout in milliseconds for reading from a server.
-#  # @default 120000
-#  #readTimeout: 120000
-
-## Detailed Query Logging
-#queryLogging:
-#  type: slf4j
-
-## Tracing configuration
-tracing:
-  # Probability, between 0.0 and 1.0, of sampling each trace.
-  probability: 0.01 # @default 0.01
-
-  # Local port to expose zpages on. Traces are accessible at http://localhost:{port}/tracez
-  #zpagesPort: 9090 # @default empty
-
-  lightstep:
-  # Either a collectorHost or grpcCollectorTarget must be defined
-  # GRPC Collector Target. Will distribute requests to all satellites returned by the DNS record
-  #grpcCollectorTarget: "dns:///lightstep-satellite.example.com:8282"
-
-  # Collector host running the lightstep satellite, will take priority over grpcCollectorTarget
-  #collectorHost: "lightstep-satellite.example.com"
-  # Collector port to be used in conjunction with the collector host
-  #collectorPort: 8282 # @default 8282
-
-  # Lightstep access token (required)
-  accessToken: "lightstep_token"
-
-  # Component name will set the "service" name in the Lightstep UI
-  # componentName: heroic # @defaults heroic
-  # Reporting interval (ms)
-  #reportingIntervalMs: 1_000 # @default 1000
-  # Max buffered spans
-  #maxBufferedSpans: 1_000 # @default 1000
-  # GRPC round robin between all hosts returned by grpcCollectorTarget
-  #grpcRoundRobin: true # @default true
-  # GRPC reset client
-  #grpcResetClient: false # @default false
+consumers:
+ - type: pubsub
+   # Topic and subscription will be created within the project if they don't exist,
+   # assuming Heroic has appropriate GCP permissions.
+   project: heroic
+   topic: metrics
+   subscription: heroic-consumer
+   schema: com.spotify.heroic.consumer.schemas.Spotify100
```

Comment:
side thought - maybe we should remove the `Spotify` from the schema with the 1.0 release? There's nothing specific to Spotify in the schema and this would allow us to have a more standard spec. Would have to do the same thing in ffwd.

Reply:
Agreed, I'll make an issue for it. We can also remove the need for the fully qualified package so make it easier to configure.

Reply:
#499

Reply:
tack!